query-engine/driver-adapters/src/proxy.rs

```diff
@@ -351,43 +353,44 @@ fn js_value_to_quaint(
             },
             serde_json::Value::Null => Ok(QuaintValue::null_character()),
             mismatch => Err(conversion_error!(
-                "expected a string in column {column_name}, found {mismatch}"
+                "expected a string in column '{column_name}', found {mismatch}"
             )),
         },
         ColumnType::Text => match json_value {
             serde_json::Value::String(s) => Ok(QuaintValue::text(s)),
             serde_json::Value::Null => Ok(QuaintValue::null_text()),
             mismatch => Err(conversion_error!(
-                "expected a string in column {column_name}, found {mismatch}"
+                "expected a string in column '{column_name}', found {mismatch}"
             )),
         },
         ColumnType::Date => match json_value {
             serde_json::Value::String(s) => NaiveDate::parse_from_str(&s, "%Y-%m-%d")
                 .map(QuaintValue::date)
-                .map_err(|_| conversion_error!("expected a date string, got {s}")),
+                .map_err(|_| conversion_error!("expected a date string in column '{column_name}', got {s}")),
             serde_json::Value::Null => Ok(QuaintValue::null_date()),
             mismatch => Err(conversion_error!(
-                "expected a string in column {column_name}, found {mismatch}"
+                "expected a string in column '{column_name}', found {mismatch}"
             )),
         },
         ColumnType::Time => match json_value {
-            serde_json::Value::String(s) => NaiveTime::parse_from_str(&s, "%H:%M:%S")
+            serde_json::Value::String(s) => NaiveTime::parse_from_str(&s, "%H:%M:%S%.f")
                 .map(QuaintValue::time)
-                .map_err(|_| conversion_error!("expected a time string, got {s}")),
+                .map_err(|_| conversion_error!("expected a time string in column '{column_name}', got {s}")),
             serde_json::Value::Null => Ok(QuaintValue::null_time()),
             mismatch => Err(conversion_error!(
-                "expected a string in column {column_name}, found {mismatch}"
+                "expected a string in column '{column_name}', found {mismatch}"
             )),
         },
         ColumnType::DateTime => match json_value {
+            // TODO: change parsing order to prefer RFC3339
```

Comment:
TODO for tech debt or later? Maybe create an issue?

Reply:
I'm actively working on this for now. So let's leave it like this and iterate. 

I also appreciate the deep review Joel! keep doing it!